src/reconstruction/rBioNet/data2model.m

```diff
@@ -138,8 +146,14 @@
 % max_msgbox = 0;
 %
 
-reactions   = rBioNetSaveLoad('load','rxn');
-metabolites = rBioNetSaveLoad('load','met');
+% if reaction and metabolite database is input manually instead of through rBioNet GUI
+if nargin>2
```

Comment:
May you use `if isfield(...)`?

Reply:
Is that really such a good idea? Then there is a "not enough input arguments" error when database is not entered and every script using this function would need to be rewritten.

Reply:
Not sure I understand ...

Reply:
Never mind,...that should be isstruct, not isfield.

Reply:
Sorry, I wanted to say: `if ~exist('database', 'var')` :smile: